### PR TITLE
ports bullet holes

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

Ports this PR: https://github.com/tgstation/tgstation/pull/30714, and the fix: https://github.com/tgstation/tgstation/pull/31233/files

Adds bullet holes to walls if they get shot at. 

https://gyazo.com/1d5b5bf433b1a66141682226883fe265

#### Changelog

:cl: Kor
rscadd: Gunfire can now leave bullet holes/dents in walls. Sprites by JStheguy.
/:cl:

### CODE REVIEW

File: code/game/turfs/closed.dm [hunk 1]
```diff
@@ -90,4 +90,17 @@
 	name = "iced rock"
 	desc = "Extremely densely-packed sheets of ice and rock, forged over the years of the harsh cold."
 	icon = 'icons/turf/walls.dmi'
-	icon_state = "icerock"
\ No newline at end of file
+	icon_state = "icerock"
+
+/turf/closed/bullet_act(obj/item/projectile/Proj)
+	..()
```
Comment (Cruix) [inline code/game/turfs/closed.dm]: This should be . = ..()

**Comment by KrabSpider** [comment]
Why? Why is this needed?

**Comment by Slinky5890** [comment]
@KrabSpider Why does there have to be an explicit reason for an interesting new addition? This doesn't mess with game balance or anything

**Comment by Super3222** [comment]
Thank you for porting this

**Comment by Kmc2000** [comment]
pretty cool actually, yeah

**Comment by ghost** [comment]
@KrabSpider "why is this needed" is a stupid question and you should stop playing if that's all you post on PRs.

in other news, cool port. But we could use a rebase :^)

**Comment by ShadowDeath6** [comment]
port the pr that lets you weld walls to remove the holes

**Pull request merged**